base/grisu.jl

```diff
@@ -13,9 +13,12 @@ include("grisu/float.jl")
 include("grisu/fastshortest.jl")
 include("grisu/fastprecision.jl")
 include("grisu/fastfixed.jl")
+include("grisu/bignums.jl")
 include("grisu/bignum.jl")
 
-function grisu(v::FloatingPoint,mode,requested_digits,buffer=DIGITS)
+const BIGNUMS = [Bignums.Bignum(),Bignums.Bignum(),Bignums.Bignum(),Bignums.Bignum()]
```

Comment:
Yeah, at the least, it's moving the two variable declarations down 10 lines into the `grisu` function.